tools/travis_watchdog.sh

```diff
@@ -166,7 +171,7 @@ print_stacktraces () {
 put_yarn_logs_to_artifacts() {
 	# Make sure to be in project root
 	cd $HERE/../
-	for file in `find ./flink-yarn-tests/target/flink-yarn-tests* -type f -name '*.log'`; do
+	for file in `find ./flink-yarn-tests/target -type f -name '*.log'`; do
```

Comment:
why is this required?

Reply:
I was debugging failing YARN tests as part of the migration. IIRC, some of the newer YARN tests log into a different directory. It doesn't hurt to search for log files to be included in the debugging file a bit more broadly.

Reply:
then let's put this into a separate commit